servers_v7.json

```diff
@@ -11,6 +11,10 @@
     "name": "SynapseOS",
     "address": ["0nera.ru:7777", "185.178.45.224:7777"]
   },
+  {
+    "name": "Voiddustry"
```

Comment:
missing ,

Reply:
how change it again?
im sorry, i not soo know github